clusterloader2/testing/load/config.yaml

```diff
@@ -48,12 +48,34 @@
 {{$mediumDeploymentsPerNamespace := SubtractInt $mediumDeploymentsPerNamespace (IfThenElse $ENABLE_JOBS 1 0)}}
 {{$bigDeploymentsPerNamespace := SubtractInt $bigDeploymentsPerNamespace (IfThenElse $ENABLE_JOBS 1 0)}}
 
+# BEGIN scheduler-throughput section
+# Min number of pods per deployment to be used for measuring scheduler throughput
+# to get enough samples and accurate measurements in small clusters.
+{{$MIN_PODS_PER_DEPLOYMENT_TO_MEASURE_SCHEDULER_THROUGHPUT := 250}}
+# Number of namespaces with scheduler throughput deployments. This is also number
+# of scheduler throughput deployments as we create 1 such deployment per namespace.
+{{$schedulerThroughputNamespaces := 2}}
+{{$schedulerThroughputPodsPerDeployment := MaxInt $MIN_PODS_PER_DEPLOYMENT_TO_MEASURE_SCHEDULER_THROUGHPUT .Nodes}}
+{{$schedulerThroughputThreshold := DefaultParam .CL2_SCHEDULER_THROUGHPUT_THRESHOLD 0}}
+# END scheduler-throughput section
+
+# BEGIN pod-startup-latency section
+# Min number of pods to be used for measuring pod startup latency to get enough
+# samples and accurate measurements in small clusters.
+{{$MIN_PODS_TO_MEASURE_STARTUP_LATENCY := 500}}
+{{$totalLatencyPods := MaxInt $MIN_PODS_TO_MEASURE_STARTUP_LATENCY .Nodes}}
+{{$latencyReplicas := DivideInt $totalLatencyPods $namespaces}}
+# END pod-startup-latency section
+
 name: load
-automanagedNamespaces: {{$namespaces}}
+automanagedNamespaces: {{AddInt $namespaces $schedulerThroughputNamespaces}}
 tuningSets:
 - name: Sequence
   parallelismLimitedLoad:
     parallelismLimit: 1
+- name: Uniform5qps
+  qpsLoad:
+    qps: 5
```

Comment:
We support 20pods/s officially - can you please add a todo to bump it (and open an issue)?

[BTW, I think we were already experimenting with it, and it's possible that it would already work]

Reply:
Oh, actually it's used also in pod-startup-latency. Adding it back and adding TODO